wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -627,12 +631,9 @@ boolean createSnapshotSync(WorkspaceRuntimeImpl runtime, String namespace, Strin
         return devMachineSnapshotFailMessage == null;
     }
 
-    private void checkWorkspaceBeforeCreatingSnapshot(WorkspaceImpl workspace) throws ConflictException {
+    private void checkWorkspaceIsRunning(WorkspaceImpl workspace, String msg) throws ConflictException {
```

Comment:
as all parameters of checkWorkspaceIsRunning method are the same except the formatted string in both calls, maybe we should only provide the "unformatted string", and keep 
workspace.getNamespace(),
workspace.getConfig().getName(),
workspace.getStatus()) at a single place (there) to avoid copy/paste